cmake/Modules/CMakeFindFrameworks.cmake

```diff
@@ -0,0 +1,32 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
+# file Copyright.txt or https://cmake.org/licensing for details.
```

Comment:
>  Can we enable 'real' comparison phase (ie one that will trigger error) on compatible systems?
Yes, but not immediately.  My understanding is that  in some cases binder relies on the standard behaviour of clang tools, which is slightly different from version to version. Therefore, for many tests there is at least one version of clang/llvm that would fail the comparison test. The solution is obvious: set the behaviour explicitly. But that would require some more time. Therefore my suggestion would be to look case-by-case on the differences and either ignore them, or, if you find the important make sure different versions of clang interpret things in the same way. 


> can we detect that we compile under CentOS-7 with Clang-3.4 and if so use comparison result as pass/fail criteria? 
Yes. CentOS7/clang3.4 support is implemented in another branch.

> Thoughts?
1) If I would decide on the infrastructure in our field, I would switch to CentOS8 some years before -> use the default version of clang on that system for comparison. 
2) In case of implementation of  "set behaviour explicitly "from the answer to the first question, it should not matter.

Reply:
> This file and others in cmake/Modules: what is the source of these files? Are they coming from Cmake tree? 
Yes.  https://github.com/Kitware/CMake/tree/master/Modules      (but not  the master)

>Can we some where note that? Possibly readme i this dir?
Typically people just copy-paste them "as is" because the files contain the references. But why not. Maybe in the general readme.

Reply:
> If I would decide on the infrastructure in our field, I would switch to CentOS8 some years before -> use the default version of clang on that system for comparison.

-- i hear you! But since HPC wold will be stuck with CentOS-7 for quite a while i am not sure if moving to CentOS-8 will make things easier for average user.